browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts

```diff
@@ -180,21 +174,13 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
         /* wantEligiblePrograms= */ [secondProgramName],
       )
 
-      await validateScreenshot(
-        page,
-        'cif-eligible-guest-confirmation-page',
-        /* fullPage= */ true,
-        /* mobileScreenshot= */ true,
-      )
       await validateAccessibility(page)
 
       await page.click('text="Apply to programs"')
 
       await validateScreenshot(
```

Comment:
**suggestion:** I think we can remove the screenshot here and just confirm that the login prompt appears.